src/components/App.tsx

```diff
@@ -17,11 +17,11 @@ const onBeforeLift = async () => {
   await refreshMembers()(store.dispatch, store.getState, undefined);
 };
 
-class AppRoot extends React.Component {
+class AppRoot extends React.Component<{}> {
   shouldRefreshOnNextForeground: boolean;
 
-  constructor() {
-    super();
+  constructor(props: {}) {
+    super(props);
```

Comment:
IT wasn't happy that we weren't passing anything to `super`, because `React.Component` expects `props` as a param.